### ISSUE TITLE
Attempting to Reorder Columns of the LoadoutGrid crashes the App

### ISSUE THREAD

**Issue by Al12rs** [Nexus-Mods/NexusMods.App#828]
# Bug Report

When attempting to reorder the horizontal order of the loadout grid columns via Drag and Drop, the App crashes.

For now the interaction was disabled to avoid crashes, but this is still a wanted functionality, so the issue should be investigated and fixed if possible.

**Comment by LukeNexusMods** [comment]
Is this still a thing?

**Comment by Al12rs** [comment]
Yes. low priority as I disabled the feature in the meantime, so no crashing. We will eventually want to enable that back but no hurry on that.
